File: Javascript/1040.js
var input = `2.0 4.0 7.5 8.0
6.4`;
let lines = input.split(/[\s]/);
let res = (lines[0] * 2 + lines[1] * 3 + lines[2] * 4 + lines[3] * 1) / 10;
console.log("Media: " + res.toFixed(1));
if (res >= 7) {
  console.log("Aluno aprovado.");
} else if (res >= 5 && res <= 6.9) {
  console.log("Aluno em exame.");
  res = (res + Number(lines[4])) / 2;
  console.log("Nota do exame: " + Number(lines[4]).toFixed(1));
  if (res >= 5) {
    console.log("Aluno aprovado.");
  } else {
    console.log("Aluno reprovado.");
  }
  console.log("Media final: " + res.toFixed(1));
} else if (res < 5) {
  console.log("Aluno reprovado.");
}

File: Python/1040.py
line1 = input().split(" ")
res = (float(line1[0]) * 2 + float(line1[1]) * 3 +
       float(line1[2]) * 4 + float(line1[3]) * 1) / 10
print('Media: %.1f' % res)
if (res >= 7):
    print("Aluno aprovado.")
elif (res >= 5 and res <= 6.9):
    print("Aluno em exame.")
    line2 = float(input())
    res = (res + line2) / 2
    print("Nota do exame: %.1f" % line2)
    if (res >= 5):
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print("Media final: %.1f" % res)
elif (res < 5):
    print("Aluno reprovado.")
